spring-cloud-stream-binder-kinesis-core/src/main/java/org/springframework/cloud/stream/binder/kinesis/properties/KinesisBinderConfigurationProperties.java

```diff
@@ -32,6 +32,12 @@
 
 	private int describeStreamRetries = 50;
 
+	private String checkpointTable = null;
+
+	private Long dynamoDbReadCapacity = 1L;
+
+	private Long dynamoDbWriteCapacity = 1L;
```

Comment:
Let's consider something like `Checkpoint` nested class! To gather all the options related to the `DynamoDbMetadataStore` configuration.

You can find sample in the https://github.com/spring-projects/spring-boot/blob/master/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/kafka/KafkaProperties.java#L71

Or take a look into the `KinesisBindingProperties`  and its `consumer` property.